### PR TITLE
Css two column layout

### PR DESCRIPTION
**Pull request by Smith-Steve**
I am not submitting this. I want to be able to look at and review the code on github while I am working on a different project, so I pushed it up. Did not know how else to do it. Sorry Cody - still requires touching up.

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Yo dude!! I know you still have this in progress, just wanted to give you a review so it's clear there are still changes needed. If you have any questions on how to tackle this, make sure you reach out in Slack!

**Comment by Smith-Steve** [comment]
@codingcodymiller Hey Cody, didn't see your comment.. Sorry about that. I wanted to look at the code as I was going along. Won't let it happen again. Sorry.

image
I think this is pretty good, I think I'm catching on in CSS a little bit.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey dude, nice progress!! Your button is not centered below the columns though, as in the example. Please get that fixed and resubmit and I will give a more thorough look

**Comment by Smith-Steve** [comment]
image

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+.item-box {
+  display: flex;
+}
+
+.buttonContainer {
+  position: absolute;
+  right: 1100px;
+  bottom: 475px;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css]: Try `text-align: center`. This should not require absolute positioning

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]
Nice work!!

**Pull request merged**